google/cloud/spanner/internal/database_admin_stub.cc

```diff
@@ -46,7 +46,7 @@ class DefaultDatabaseAdminStub : public DatabaseAdminStub {
     grpc::Status status =
         database_admin_->CreateDatabase(&client_context, request, &response);
     if (!status.ok()) {
-      return google::cloud::grpc_utils::MakeStatusFromRpcError(status);
+      return MakeStatusFromRpcError(status);
```

Comment:
FYI, these unqualified calls to `MakeStatusFromRpcError()` will use ADL. That may be OK. but qualifying them with `google::cloud::` would avoid the ADL and make their location clear. It was previously not using ADL. Not a huge deal either way.

I'll let you decide if you'd like to change anything.